tests/cpp_integration_tests/test_math_solver.cpp

```diff
@@ -196,7 +196,7 @@ TEST_CASE("Test math solver") {
     DoubleComplex const yref = 10.0 - 50.0i;
     pf_input.source = {vref};
     // source param and result
-    param.source_param = {yref};
+    param.source_param = {SourceCalcParam{yref, yref}};
```

Comment:
Here and below: my blind wonder without knowing anything, wouldn't it suffice to simply do
```suggestion
    param.source_param = SourceCalcParam{yref, yref};
```

Reply:
`{SourceCalcParam{yref, yref}}` is needed because `param.source_param` is of type `std::vector<power_grid_model::SourceCalcParam>`

Reply:
Then a bad naming detected ;)